src/views/virtualmachines/creation-wizard-new/steps/DeploymentDetailsStep/components/CreationMethodTileGroup/CreationMethodTileGroup.tsx

```
@@ -24,7 +26,9 @@ const CreationMethodTileGroup: FC = () => {
             <CreationMethodTile
               creationMethod={method}
               isChecked={creationMethod === method}
-              setSelectedCreationMethod={setCreationMethod}
+              setSelectedCreationMethod={(creationMethod) =>
+                setValue('vmData.creationMethod', creationMethod)
+              }
```

ESLint

[error] Expected "setSelectedCreationMethod" to come before "isChecked"

(perfectionist/sort-jsx-props)